node-tests/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
```

Comment:
we are not using ES6 modules in this folder, so this might be the wrong configuration here.